torch_xla/csrc/tensor_impl.h

```diff
@@ -31,6 +31,7 @@ class XLATensorImpl : public c10::TensorImpl {
   void shallow_copy_from(const c10::intrusive_ptr<TensorImpl>& impl) override;
 
   at::IntArrayRef sizes_custom() const override;
+  c10::SymIntArrayRef sym_sizes() const override;
```

Comment:
@Krovatkin looks I no longer see the `virtual c10::SymIntArrayRef sym_sizes()` [here](https://github.com/pytorch/pytorch/blob/master/torch/csrc/lazy/core/tensor_impl.h#L46). Is that a final call or do you plan to add it back?